pynucastro/rates/derived_rate.py

```diff
@@ -51,6 +54,12 @@ def __init__(self, rate, compute_Q=False, use_pf=False):
         if not all(nuc.spin_states for nuc in self.rate.products):
             raise ValueError(f'One of the products spin ground state ({self.rate.products}), is not defined')
 
+        if not use_unreliable_spins:
```

Comment:
I wanted to write a test to cover this logic branch, but all "experimentally weak" spin values in our data only show up in tabular libraries or in weak rates. Therefore: The following code will never be executed with the data we have currently, and DerivedRate will never actually care if you pass in use_unreliable_spins as True or False.

Reply:
Alternatively.. I could write a test over all reaclib rates confirming this assumption, that even with use_unreliable_spins=False this particular ValueError is never raised. Or I could write fake data for the spin table and test it that way.